### ISSUE TITLE
Visualizing rotation lists in an inverse pole figure

### ISSUE THREAD

**Issue by din14970** [pyxem/orix#122]
Not sure if this is the place to ask, but I would like to get a quick visual on the rotations generated by the functions in `rotation_list_generators`, as shown in pyxem/diffsims#103 . Is there already some kind of IPF plot function in diffsims/orix, or is this being considered? Right now, I feel like I've created a very crappy workaround by creating a fictitious ebsd map from the orientations, which I then import into MTEX:

```python
rot_list_cubic = diffsims.generators.rotation_list_generators.get_fundamental_zone_grid(225, 10)
euler = np.array(rot_list_cubic)
ln = euler.shape[0]
phase_id = np.ones((ln, 1))
score_xy = np.vstack([np.ones(ln), range(ln), np.ones(ln)]).T
results_array = np.hstack([phase_id, euler, score_xy])
np.savetxt("rotlist.txt", results_array, delimiter="\t", newline="\r\n")
``` 

Also, do you have (or would you consider opening) a Gitter channel for pyxem/orix/diffsims? I feel like I have questions that are more discussion type, not really feature requests or bug reports.

**Comment by hakonanes** [comment]
@din14970, thanks a lot for opening this issue, which helps raise the PF/IPF plotting on the agenda. My short answer though is that using MTEX is the only option at the moment... If you just want to import rotations/orientations in MTEX, I don't think you need the phase and auxiliary property columns, just the Euler angles: https://mtex-toolbox.github.io/OrientationImport.html.

One alternative option, @pc494, we should be able to plot the fundamental zone to visualize rotation sampling? As in https://nbviewer.jupyter.org/github/pyxem/orix-demos/blob/master/02%20-%20Clustering%20Orientations.ipynb#vis. I haven't used that myself, though, so I cannot say.

As @pc494, I'm not keen on Gitter, because I think discussing something in a GitHub issue makes it easier to follow and keep track of, thus hopefully shortening the time from an idea to a PR.

**Comment by din14970** [comment]
Alright, yes I figured orix is still very new so the option might not be there, but I thought I'd ask. I hope there will be a day where I can ditch Matlab altogether.

Maybe now your userbase is still small so I can see your perspective on gitter. But I see a lot of growth potential for these tools. At some point you will get users asking the same questions over and over again (which I try to avoid but it happens), and then I think you may want to de-clutter your issues and relegate "tech support" discussions to elsewhere.

**Comment by din14970** [comment]
> If you just want to import rotations/orientations in MTEX, I don't think you need the phase and auxiliary property columns, just the Euler angles: https://mtex-toolbox.github.io/OrientationImport.html.

Also thanks for this tip, I figured it would be easiest to follow the`save_mtex_map` function, but euler angles are indeed sufficient.

**Comment by pc494** [comment]
I agree with everything that's been said here. I'll leave this issue open until the feature is added.

**Comment by hakonanes** [comment]
Inverse pole figure plotting can be implemented as an extension of the `StereographicPlot`. This is something I think shouldn't be too difficult, and will work towards implementing myself, by trying to utilize similar functionality as is done for [Matplotlib's polar projection](https://matplotlib.org/stable/gallery/pie_and_polar_charts/polar_scatter.html#scatter-plot-on-polar-axis-confined-to-a-sector). My goal is to be able to define a triangle spine with three vectors, and plot only vectors inside this triangle.

**Comment by hakonanes** [comment]
I anticipate that this can be closed after a follow-up PR to #232.
